src/olcs/contrib/manager.js

```diff
@@ -335,4 +335,14 @@ olcs.contrib.Manager = class {
       });
     });
   }
+
+  /**
+   * @protected
+   * @return {Cesium.Rectangle|undefined}
+   */
+  getCameraExtentRectangle() {
```

Comment:
why not using a getter `get cameraExtentRectangle` ?

Reply:
I tried on another class and got an error from GCC. (something like export is only allowed for global methods and properties).

Reply:
ok